docs/src/content/docs/ru/components/card-grids.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: Карточные сетки
+description: Узнайте, как обернуть несколько карточек в сетку в Starlight.
+sidebar:
+  order: 4
+---
+
+import { Card, CardGrid, LinkCard } from '@astrojs/starlight/components';
+
+Чтобы обернуть несколько компонентов [`<Card>`](/ru/components/cards/) или [`<LinkCard>`](/ru/components/link-cards/) в сетку, используйте компонент `<CardGrid>`.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<CardGrid slot="preview">
+	<Card title="Stars" icon="star">
```

Comment:
```suggestion
	<Card title="Звёзды" icon="star">
```

Could we translate both card titles? I assume it would make sense as some other pages have these examples translated?